### PR TITLE
workflows: bump invenio-classifier to version ~1.1.0

### LINKED ISSUE #2328: invenio-classifier generates non-indexable fields
e.g. https://labs.inspirehep.net/api/holdingpen/632571 contains `interfaces.` (note the extra `.` at the end) among `#/_extra_data/classifier_results/complete_output` which makes ElasticSearch fail (as explained by @jacquerie ).
This happens despite `interfaces.` not being present in the HEPont ontology.

@ksachs @kaplun, maybe you have some idea of how this could arise?

### PR DESCRIPTION
**Pull request by jacquerie**
Closes #2328 
Closes #2347

### CODE REVIEW

File: tests/unit/workflows/test_workflows_tasks_actions.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def _get_auto_reject_obj(decision, has_core_keywords):
     obj_params = {
         'classifier_results': {
             'complete_output': {
-                'Core keywords': ['something'] if has_core_keywords else [],
+                'core_keywords': ['something'] if has_core_keywords else [],
```
Comment (david-caro) [inline tests/unit/workflows/test_workflows_tasks_actions.py]: this is way nicer :)

**Comment by david-caro** [comment]
This is not related though with the autoreject issue right? (I'm playing with similar files....)

**Comment by jacquerie** [comment]
> This is not related though with the autoreject issue right? (I'm playing with similar files....)

It might. I haven't investigated that issue...

**Comment by jacquerie** [comment]
@kaplun: I created a new PR because I wasn't able to push to the branch in #2347...

**Comment by kaplun** [comment]
Indeed the flag was turned off: mmh I need to authorize explicitly every PR? 

On another hand: note that accessing old workflows might fail due to the restructuring of the json (or very likely it will simply not display keywords).

File: tests/unit/workflows/test_workflows_tasks_actions.py (hunk 1 above)
Comment (kaplun) [inline tests/unit/workflows/test_workflows_tasks_actions.py]: eheh, I changed all around. But of course it will blown up existing records.

**Comment by jacquerie** [comment]
> (or very likely it will simply not display keywords).

Angular's templating is forgiving: if you don't have the key, it's not going to blow up.

**Pull request merged**